pkg/hubble/peer/types/client.go

```diff
@@ -50,13 +49,10 @@ type LocalClientBuilder struct {
 
 // Client implements ClientBuilder.Client.
 func (b LocalClientBuilder) Client(target string) (Client, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), b.DialTimeout)
```

Comment:
As far as I can tell the `DialTimeout` is not completely unused, so IMO we can remove it.